frontend/src/styles/scss/custom.scss

```diff
@@ -1,18 +1,15 @@
-
 // Your variable overrides
 // $body-bg: #000;
-$body-bg:  rgb(194, 239, 255);
+$body-bg : rgb(194, 239, 255);
```

Comment:
lol typo?

Reply:
auto formatting be like that sometimes 🤷‍♀️